test/models/document_test.rb

```diff
@@ -6,15 +6,15 @@ class DocumentTest < ActiveSupport::TestCase
   end
 
   test "has an associated assistant" do
-    assert_instance_of Assistant, documents(:cat_photo).assistant
+    assert_instance_of Assistant, documents(:background).assistant
   end
 
   test "has an associated message" do
     assert_instance_of Message, documents(:cat_photo).message
   end
 
-  test "simple create works" do
-    assert_nothing_raised do
+  test "create fails without a file" do
+    assert_raises do
```

Comment:
I think assert_raises can take the expected Error class as a parameter -- adding that here will prevent false-passes on this test in the future if it starts throwing StandardError for some reason.

Reply:
Good call, fixing